src/components/notifications/SMSC.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <div class="mb-3">
+        <label for="smsc-login" class="form-label">{{ $t("API Username") }}</label>
+        <i18n-t tag="div" class="form-text" keypath="wayToGetClickSendSMSToken">
```

Comment:
All the new translations need to be added to `en.json`. Otherwise this is not going to be translatable into other languages.